common/JackServerGlobals.cpp

```diff
@@ -195,7 +195,7 @@ bool JackServerGlobals::Init()
         optind = 1; // Important : to reset argv parsing
 
         while (!master_driver_name &&
-                (opt = getopt_long(argc, argv, options, long_options, &option_index)) != EOF) {
+                (opt = getopt_long(argc, argv, options, long_options, &optind)) != EOF) {
```

Comment:
This one seems radical and would deserve it's own pull request. I don't understand what happens though. Can you elaborate what's happening here?

Reply:
I'll revert this! Sorry!
Was struggling with externs in getopts